### PR TITLE
fix(visualizations): Update Vis Components directory structure and package files

### PR DESCRIPTION
**Pull request by dbchristopher**
- Add file structure and build files for new `@looker/visualizations-adapters` package
- Update all visualization package.json files to match dependency changes in our internal repo

### CODE REVIEW

**Comment by dbchristopher** [comment]
@lukelooker here is the updated dependency graph that matches the changes in our internal repo

**Comment by dbchristopher** [comment]
@lukelooker do you think this is worth merging for now just to resolve those build errors?

File: packages/visualizations-nivo/package.json [hunk 1]
```diff
@@ -2,23 +2,25 @@
  "name": "@looker/visualizations-nivo",
  "main": "src/index.ts",
  "version": "0.1.0",
- "private": true,
+ "publishConfig": {
+  "access": "restricted"
+ },
```
Comment (mdodgelooker) [inline packages/visualizations-nivo/package.json:7]: I'm not familiar with how this is treated differently from `"private": true,` but trusting it's what we need for these packages. 🙂

Comment (mdodgelooker) [reply packages/visualizations-nivo/package.json:7]: Side note, should this config be added to the "Facilitating Pre-Launch Packages" doc?

**Review by mdodgelooker — APPROVED** [review]
LGTM!

File: packages/visualizations-nivo/package.json (hunk 1 above)
Comment (lukelooker) [reply packages/visualizations-nivo/package.json:7]: This is going to cause trouble with publishing all the other packages. Within the OSS repo we need to keep "private": true in place so we can not publish these.

**Review by lukelooker — CHANGES_REQUESTED** [review]

File: packages/visualizations-nivo/package.json (hunk 1 above)
Comment (mdodgelooker) [reply packages/visualizations-nivo/package.json:7]: Oh good to know, thanks @lukelooker !

**Comment by lukelooker** [comment]
I'm wondering if we should just pull all of these packages out of the OSS repo for right now? It seems like keeping all of that within a `launch-vis` branch in the prerelease repo would leave this more "self-serve" for now and reduce the number of "hops" needed to make an update.

File: packages/visualizations-nivo/package.json (hunk 1 above)
Comment (dbchristopher) [reply packages/visualizations-nivo/package.json:7]: oh thanks for flagging that! I think I misunderstood some feedback from a previous PR

**Comment by dbchristopher** [comment]
Closing in favor of #2570

**Pull request closed without merging**